independent-publisher-connectors/AfterShip/apiDefinition.swagger.json

```diff
@@ -0,0 +1,2744 @@
+                          "description": "Created date and time"
+                        },
+                        "checkpoint_time": {
+                          "type": "string",
+                          "description": "Time reached checkpoint"
+                        },
+                        "city": {
+                          "type": "string",
+                          "description": "City"
+                        },
+                        "coordinates": {
+                          "type": "array",
+                          "items": {},
+                          "description": "Coordinates"
+                        },
+                        "country_iso3": {
+                          "type": "string",
+                          "description": "Country code in ISO 3 format"
+                        },
+                        "country_name": {
+                          "type": "string",
+                          "description": "Country name"
+                        },
+                        "message": {
+                          "type": "string",
+                          "description": "Message"
+                        },
+                        "state": {
+                          "type": "string",
+                          "description": "State"
+                        },
+                        "tag": {
+                          "type": "string",
+                          "description": "Tags"
+                        },
+                        "subtag": {
+                          "type": "string",
+                          "description": "Sub tags"
+                        },
+                        "subtag_message": {
+                          "type": "string",
+                          "description": "Sub tag messages"
+                        },
+                        "zip": {
+                          "type": "string",
+                          "description": "Zip code"
+                        },
+                        "raw_tag": {
+                          "type": "string",
+                          "description": "Raw tags"
+                        }
+                      },
+                      "description": "checkpoint"
+                    }
+                  },
+                  "description": "data"
+                }
+              }
+            }
+          }
+        }
+      }
+    }
+  },
+  "definitions": {},
+  "parameters": {},
+  "responses": {},
+  "securityDefinitions": {
+    "api_key": {
+      "type": "apiKey",
+      "in": "header",
+      "name": "aftership-api-key"
+    }
+  },
+  "security": [],
+  "tags": [],
+  "x-ms-connector-metadata": [
+    {
+      "propertyName": "Website",
+      "propertyValue": "https://www.aftership.com"
+    },
+    {
+      "propertyName": "Privacy policy",
+      "propertyValue": "https://www.automizely.com/legal/privacy"
+    },
+    {
+      "propertyName": "Categories",
+      "propertyValue": "Shipment;Logistics"
```

Comment:
@taiki-yoshida  The connector category is invalid. You can only use maximum of two categories from the predefined list: AI, Business Management, Business Intelligence, Collaboration, Commerce, Communication, Content and Files, Finance, Data, Human Resources, Internet of Things, IT Operations, Lifestyle and Entertainment, Marketing, Productivity, Sales and CRM, Security, Social Media, Website

Docs - [Add metatda](https://docs.microsoft.com/en-us/connectors/custom-connectors/certification-submission#step-3b-product-or-end-service-metadata)

Reply:
Fixed to commerce